cdap-standalone/src/test/java/co/cask/cdap/StandaloneTester.java

```diff
@@ -47,10 +45,14 @@
   private static final Logger LOG = LoggerFactory.getLogger(StandaloneTester.class);
 
   private final TemporaryFolder tmpFolder = new TemporaryFolder();
+  private final Object [] configs;
   private CConfiguration cConf;
   private StandaloneMain standaloneMain;
 
-  public StandaloneTester() {
+  public StandaloneTester(Object ... configs) {
```

Comment:
Followed the same convention as in ``TestConfiguration``. I think the idea is that callers don't have to convert values to strings before passing it here. They are converted inside here.